### ISSUE TITLE
Applet about dialogs not showing translated text.

### ISSUE THREAD

**Issue by fredcw** [linuxmint/cinnamon#9482]
```
 * Cinnamon version 4.4 
 * Distribution - Mint 19.3 (64-bit)
 
 ```
**Issue**
The about dialogs of cinnamon 3rd party applets show only English strings even though translated strings are present in the applet's .po files

**Steps to reproduce**
This happens on linux Mint 19.3 but not in manjaro. I haven't tested mint 20.

`cinnamon-json-makepot -i` makes no difference.

https://i.imgur.com/7rwqdNk.jpg

I not sure if this is the right repository?

**Comment by Odyseus** [comment]
Hello, @fredcw.

This issue was already fixed by #9236 and is part of Cinnamon version 4.6.x (Linux Mint 20 and Manjaro).

**Comment by fredcw** [comment]
@Odyseus Ok, thanks.
